resources/custom-elements-helpers.js

```diff
@@ -0,0 +1,262 @@
+function create_window_in_test(t, srcdoc) {
+  let p = new Promise((resolve) => {
+    let f = document.createElement('iframe');
+    f.srcdoc = srcdoc ? srcdoc : '';
+    f.onload = (event) => {
+      let w = f.contentWindow;
+      t.add_cleanup(() => f.parentNode && f.remove());
```

Comment:
Note that the check for `f.parentNode()` is not needed; `remove()` is a no-op in that case.

Reply:
removed

Reply:
https://travis-ci.org/web-platform-tests/wpt/jobs/458915986#L632 all PASS on chrome now.